eng/versioning/pom_file_version_scanner.ps1

```diff
@@ -558,25 +589,18 @@ Get-ChildItem -Path $Path -Filter pom*.xml -Recurse -File | ForEach-Object {
         return
     }
 
-    if ($PomFilesIgnoreParent -contains $pomFile)
+    $xmlPomFile = New-Object xml
+    $xmlPomFile.Load($pomFile)
+    if ($ValidParents -notcontains $xmlPomFile.project.parent.artifactId)
```

Comment:
I think we should probably scan the parent POMs also. I did this in the branch I was working on:
```
+    if ($ValidParents -notcontains $xmlPomFile.project.parent.artifactId -and $ValidParents -notcontains $xmlPomFile.project.artifactId)
```

Reply:
@billwert, funny is that this is the same change I'd made locally and will be up with the next commit. Just an FYI the only parent this isn't currently scanning is [sdk/parents/azure-sdk-parent/pom.xml](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/parents/azure-sdk-parent/pom.xml) which is effectively the grandparent and, itself, doesn't have a parent. All of the other ValidParents, aside from azure-sdk-parent,
`$ValidParents = ("azure-sdk-parent", "azure-client-sdk-parent", "azure-data-sdk-parent", "azure-perf-test-parent", "azure-cosmos-spark_3_2-12")`
were getting scanned. In the case of azure-client-sdk-parent and azure-data-sdk-parent, their parent is azure-sdk-parent, and for azure-perf-test-parent and azure-cosmos-spark_3_2-12 their parent is azure-client-sdk-parent.